### ISSUE TITLE
Intransitive card game. 

### ISSUE THREAD

**Issue by Carreau** [Quansight/scipy-2022-swag#2]
There are games know as "intransitive", basically A > B > C > A, so thre can't be any "winner". It would be good to figure out how to make this automatically for trading cards after giving them statistics, like attack/defence.

**Comment by Carreau** [comment]
Talking with @danielballan, he suggested putting the cad on a cycle, and finding  way to count the number of nodes to go from A to B in clockwise and anticlockwise direction, and the smallest one give us which card wins.

**Comment by Carreau** [comment]
There might be also some things we can use out of dobble game that uses projective plane math to make sure two cards only have a single symbol in common.

**Comment by danielballan** [comment]
I guess rock--paper-scissors is an example of such a game. One useful train of thought may be to consider how to extend that to N objects in a graphical way.

**Comment by Carreau** [comment]
what about vector product if two object are on a 2D plane ?

**Comment by danielballan** [comment]
Yeah, that seems doable mentally. How does it determine which card “wins”?

**Comment by Carreau** [comment]
> How does it determine which card “wins”?

Whatever is positive. As A^B = - B^A as long as you have a convention to consider your card first that should be good.

**Comment by danielballan** [comment]
That makes sense!

If each card gets a 2D coordinate there are several games that might be possible: battleship is one.
